### PR TITLE
Mayhem integration

### PR DESCRIPTION
**Pull request by prateekvishnu**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Namespaced project name that the target belongs to
+project: orgize
+
+# Target name (should be unique within the project)
+target: orgize
```
Comment (xansec) [inline Mayhemfile]: I understand that the existing fuzz target is called `fuzz_target_1`. Still, looking at the target, it looks like the target is fuzzing the Org.parse() function. Please update the target name accordingly (you may also update the name of the target if you choose, perhaps upstream would even appreciate it).

**Review by xansec — CHANGES_REQUESTED** [review]
On an unrelated note, it looks like the test cases per second are very low. This doesn't seem to be a fault in the harness, but the code itself. It might help to provide a seed corpus (something like `* DONE Title :tag:`) to see if this improves parse speed. It may also simply be a very sensitive target. https://mayhem.forallsecure.com/prateekvishnu/orgize/orgize/2

**Comment by prateekvishnu** [comment]
I am a little unsure about the seed corpus — resolved other issues.

**Comment by xansec** [comment]
Added seed. LGTM!

**Pull request merged**